roles/django/defaults/main.yml

```diff
@@ -33,4 +33,5 @@ django_csp_enforce_uri: http://example.com/
 
 # media root owner
 media_user: "www-data"
-media_group: "{{ media_user }}"
\ No newline at end of file
+media_group: "{{ media_user }}"
+media_root: "/var/www/media/"
```

Comment:
we have the same default path set in multiple places; the other places weren't getting loaded in this playbook. suggestions for a better solution?

Reply:
The role defaults file is a great place for default values that should apply across environments and servers. Just remember that any other value for that variable will override the role default.